authorizer-app-backend/src/main/java/org/radarbase/authorizer/doa/entity/RestSourceUser.kt

```diff
@@ -29,6 +29,10 @@ class RestSourceUser {
     @SequenceGenerator(name = "sequenceGenerator", sequenceName = "rest_source_user_id_seq", initialValue = 1, allocationSize = 1)
     var id: Long? = null
 
+    // Date when the user is created
+    @Column(name = "created_at")
+    var createdAt: Instant = Instant.now()
```

Comment:
Since nullable constraint is set to true in DB, you should make this var nullable too.